cont.c

```diff
@@ -849,6 +849,12 @@ NOINLINE(static VALUE cont_capture(volatile int *volatile stat));
         if (!(th)->ec->tag) rb_raise(rb_eThreadError, "not running thread"); \
     } while (0)
 
+rb_thread_t*
+fiber_thread_ptr(const rb_fiber_t *fiber)
```

Comment:
```suggestion
rb_fiber_threadptr(const rb_fiber_t *fiber)
```

Reply:
It has to be prefixed, and `threadptr` is used in the other places.

Reply:
Thanks for the fixes :)